src/javascript/app_2/Stores/Modules/Trading/trade_store.js

```diff
@@ -396,6 +416,25 @@ export default class TradeStore extends BaseStore {
         }
     }
 
+    @action.bound
+    changeAllowEquals() {
+        if (['rise_fall', 'rise_fall_equal'].includes(this.contract_type)) {
+            this.is_allow_equal = false;
+
+            if (this.symbol[0] === 'R') {
```

Comment:
what is this?

Reply:
i am actually checking whether the symbol is volatility indices